packages/client/components/AddTeamDialog.tsx

```diff
@@ -33,15 +28,16 @@ interface Props {
 const AddTeamDialogViewerFragment = graphql`
   fragment AddTeamDialog_viewer on User {
     ...AdhocTeamMultiSelect_viewer
-    organizations {
+    organization(orgId: $orgId) {
```

Comment:
Refactored this component to query the org rather than all organizations. This modal only appears in `orgTeams` which belongs to one org.

I think the component may have originally been used elsewhere for a feature that has since been removed. 

Querying one org is more efficient and removes the `mutualOrgIds` logic which is hard to follow.

Reply:
-1 `AdhocTeamMultiSelect` still selects all orgs and adds completion for users from all orgs. Are we sure we want this behaviour?

Reply:
True, I can update this to just query the same org, too